packages/gatsby-source-kontent/tests/__snapshots__/createSchemaCustomization.items.spec.ts.snap

```diff
@@ -128,6 +129,7 @@ type kontent_item_rich_text_element_image {
   description: String
   height: Int!
```

Comment:
Width and height should not be required. This query fails if you want to load SVG from a rich text element:

build error:

```
 Cannot return null for non-nullable field kontent_item_rich_text_element_image.width.
4:37:37 PM: The field "kontent_item_rich_text_element_image.width." was explicitly defined as non-nullable via the schema customization API (by yourself or a plugin/theme). This means that this field is not optional and you have to define a value. If this is not your desired behavior and you defined the schema yourself, go to "createTypes" in gatsby-node.js. If you're using a plugin/theme, you can learn more here on how to fix field types:
https://www.gatsbyjs.org/docs/schema-customization/#fixing-field-types
```

Reply:
I reckon this is not a bug. Moreover, it just does not make sense - the essence of the matter in using `ImageElement` is an automatic generation and applying underlying image transformations and query string params - this makes no sense with vector format, right? 

Besides that, SVGs work just fine with the new image element if you specify `width` and ` height` params/props.

As I see it, the mandatory `width` and `height` are just a part of the underlying `gatsby-image-plugin`. contract, Therefore, if you'd like to use the image plugin, you have to follow a mandatory contract.
On the other hand, if you are dealing with SVGs and you don't know dimensions or you don't want to use ImageElement you can use the `<img>` tag and everything works as usual in js/HTML/react.

What do you think?

Reply:
To clarify - the error is in the Kontent Gatsby source plugin.

Exact root cause is here:
https://github.com/Kentico/kontent-gatsby-packages/blob/master/packages/gatsby-source-kontent/src/template.items.schema.gql#L111-L112

Tests need to regenerate snapshot after fix.

Reply:
#178 is merged, so the fix could be cherry-picked here as well